ams/api/pom.xml

```diff
@@ -25,7 +25,7 @@
     <parent>
         <groupId>com.netease.amoro</groupId>
         <artifactId>amoro-ams</artifactId>
-        <version>0.5.0-SNAPSHOT</version>
+        <version>0.5.1-SNAPSHOT</version>
```

Comment:
May not change to 0.5.1

Reply:
OK, I will set it back to 0.5.0-SNAPSHOT